src/policy/mod.rs

```diff
@@ -4,13 +4,16 @@
 
 use config_select::select_env;
 
+use crate::host::MiralisContext;
 use crate::virt::VirtContext;
 
 mod default;
 mod keystone;
+mod protectpayload;
```

Comment:
Small nit, but for naming consistency let's use `protect_payload` with an underscore (`_`)